rasa/utils/tensorflow/environment.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import os
+from typing import Text, Dict
+import typing
+
+if typing.TYPE_CHECKING:
```

Comment:
optional: move this below the last "normal" import (line 15)
```suggestion
if typing.TYPE_CHECKING:
```